test_elasticsearch/test_dsl/_async/test_document.py

```diff
@@ -582,21 +582,21 @@ def test_meta_fields_can_be_set_directly_in_init() -> None:
     assert md.meta.id is p
 
 
-@pytest.mark.asyncio
+@pytest.mark.anyio
```

Comment:
Could these have been removed, and `anyio_mode = "auto"` added to `[tool.pytest.ini_options]` in `pyproject.toml`?

Reply:
No, because some tests are only compatible with asyncio, in which case we stick with `pytest.mark.asyncio`. More generally, I like the explicit markers

Reply:
Good to know, thanks! Not going to rewrite everything now but I'll keep it in mind.